### PR TITLE
move WORKSPACE to build/ so it doesn't require root approval

### PR DESCRIPTION
**Pull request by mikedanese**
RFC. What do you think?

### CODE REVIEW

**Comment by mikedanese** [comment]
We could do the same for BUILD.bazel.

File: build/workspace.bzl [hunk 1]
```diff
@@ -0,0 +1,73 @@
+# only native extensions are allowed in this file. For repository
+# initialization use :repositories.bzl. This is to avoid circular
```
Comment (ixdy) [inline build/workspace.bzl]: that part is a little confusing, though I'm not sure there's much of a way around it.

**Review by ixdy — COMMENTED** [review]
This is clever.

I wonder how much work it'd be to add per-file approvers to the mungebot. I'm guessing more work than this.

We can alias most of the stuff in `BUILD.bazel` using aliases rather than skylark probably, right?

**Comment by spxtr** [comment]
I'd rather just add you or ixdy to top-level OWNERS.

**Comment by cblecker** [comment]
There's also https://github.com/kubernetes/test-infra/pull/1789 to exempt files from OWNER approvals, but that may not be a good idea either.

**Pull request closed without merging**